x-pack/plugins/enterprise_search/public/applications/app_search/components/curations/views/curation_suggestion/curation_suggestion_logic.ts

```diff
@@ -79,7 +79,8 @@ export const CurationSuggestionLogic = kea<
       const { engineName } = EngineLogic.values;
 
       try {
-        const suggestionResponse = await http.get(
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        const suggestionResponse = await http.get<any>(
```

Comment:
What was your intent here? It seems better to omit the type rather than add `any` and a disable.

Reply:
Oh that's weird, this page was showing me a diff that made it appear like you added this, but now it's not. Disregard this comment.

Reply:
The typescript stuff came from https://github.com/elastic/kibana/pull/114265 but it causes a merge conflict with the new path that I had to resolve, so I think it got confused.